.travis.yml

```diff
@@ -15,7 +15,9 @@ matrix:
         - php: hhvm
 
 before_script:
-    - echo 'extension=mongo.so' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+    # The mongo extension is not yet available on PHP 7
+    - if [[ "$TRAVIS_PHP_VERSION" != "7.0" ]] && [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then echo 'extension=mongo.so' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
```

Comment:
well, this make it easier to change the condition once one of them becomes compatible with mongo